### DISCUSSION TITLE
Access points (all of them)

### DISCUSSION THREAD

**Discussion by gerontakos** [crystalyragui/MARC2RDA#381] [General]
We know we'll always want access points for some RDA entities. I see we already have a way to create access points for manifestations (see [discussion #376](https://github.com/uwlib-cams/MARC2RDA/discussions/376)). That's a perfect example of what I'm hoping for regarding other entities. What else do we also want access points for? I expect we want access points for all works, given that some works do not have titles. If we can devise the access points we need and the MARC data that will create those access points, we can code it now for the November presentation.

**Comment by CECSpecialistI** [comment]
If human catalogers are working with the data, it seems they will want access points for Works, Expressions, and Manifestations, and to be able to use preferred labels as access points for all other RDA entities.
